include/CL/sycl/math.hpp

```diff
@@ -0,0 +1,159 @@
+#ifndef TRISYCL_SYCL_MATH_HPP
+#define TRISYCL_SYCL_MATH_HPP
+
+/** \file
+    Implement a wrapper to OpenCL math operations
+    Joan.Thibault AT ens-rennes POINT fr
+    This file is distributed under the University of Illinois Open Source
+    License. See LICENSE.TXT for details.
+*/
+
+#include <cmath>
+
+namespace cl {
+namespace sycl {
+  #define wrap(FUN) template<typename T> T FUN(T x) {return std::FUN(x);}
```

Comment:
We do not indent in the namespaces.
Furthermore the #define are not usually indented.
To avoid conflicts with some user names, try more complex macro names, such as TRISYCL_WRAP3, etc.
You can do a #undef TRISYCL_WRAP3 at the end of the file when no longer used.